src/components/v2/V2Project/V2FundingCycleSection/PayoutSplitsCard.tsx

```diff
@@ -16,7 +16,11 @@ import { useETHPaymentTerminalFee } from 'hooks/v2/contractReader/ETHPaymentTerm
 
 import DistributePayoutsModal from './modals/DistributePayoutsModal'
 
-export default function PayoutSplitsCard() {
+export default function PayoutSplitsCard({
+  hideDistributeBtn,
```

Comment:
**nitpick** kinda prefer full words for stuff for max readability and min confusion, wdyt:

```suggestion
  hideDistributeButton,
```